src/Confluent.Kafka.Avro/AvroSerializer.cs

```diff
@@ -53,13 +53,13 @@ public class AvroSerializer<T> : ISerializer<T>
         private const string AutoRegisterSchemaPropertyName = "avro.serializer.auto.register.schemas";
 
         /// <summary>
-        ///     The SchemaId corresponding to type <see cref="T"/>, or null if 
+        ///     The SchemaId corresponding to type <typeparamref name="T"/>, or null if
         ///     the serializer has yet to be invoked.
         /// </summary>
         public int? SchemaId { get; private set; } = null;
 
         /// <summary>
-        ///	    The <see cref="ISchemaRegistryClient"/> instance used for communication
+        ///	    The ISchemaRegistryClient instance used for communication
```

Comment:
it would be better to have the reference, but it was causing compile warnings and I couldn't figure it out.